cbt-bot/proto/cbt_bot.proto

```
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package cbt_bot;
```

Buf (1.47.2)

Files with package "cbt_bot" must be within a directory "cbt_bot" relative to root but were in directory "cbt-bot/proto".

(PACKAGE_DIRECTORY_MATCH)